### PR TITLE
add pip install packages

### LINKED ISSUE #24: Add setup instructions for local installation
Need to list all the python packages that are required to be installed to follow the tutorial

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #24

### CODE REVIEW

**Comment by amanmdesai** [comment]
@klieret hopefully this should fix the setup instructions. Let me know if some fixes are required

File: setup.md [hunk 1]
```diff
@@ -21,13 +21,25 @@ to get a JupyterLab session with everything included. Once JupyterLab opens, cli
 If you want to install some of these packages on your own or your lab's computer, I recommend [Miniforge](https://github.com/conda-forge/miniforge) (or Anaconda/Miniconda with the [conda-forge channel](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge)). This method also provides a way to [install ROOT in the same environment](https://github.com/conda-forge/root-feedstock#readme). After setting that up, Uproot (for instance) could be installed as
 
 ~~~
-conda install uproot awkward   # ... others?
+conda install numpy uproot4 awkward hist particle hepunits matplotlib boost-histogram iminuit zfit vector fastjet
 ~~~
 {: .language-bash}
 
 If one of the above-mentioned packages is not available on conda-forge (yet), it can be installed in the same environment using pip:
```
Comment (klieret) [inline setup.md:28]: The conda file also takes care of these packages (see the `pip` bit at the bottom). So I would delete everything about here and just write "Alternatively, you can install all required packages locally with `pip`: Take a look at `environment.yml` for a list (this includes all dependencies listed under the `pip` key and some of the dependencies listed above)."

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by klieret** [comment]
Sorry for taking this in another direction!

**Comment by amanmdesai** [comment]
Hi @klieret  I have updated the setup as per your suggestions, could you please re-review?

**Review by klieret — APPROVED** [review]
Thanks!

**Pull request merged**